app/models/region.rb

```diff
@@ -54,6 +54,36 @@ def log_payload
     attrs.symbolize_keys
   end
 
+  def syncable_patients
+    case region_type
+      when "block"
+        registered_patients
+          .union(assigned_patients)
+          .union(appointed_patients)
+      else
+        registered_patients
+    end
+  end
+
+  def registered_patients
+    Patient
+      .with_discarded
```

Comment:
Callers should be responsible for adding `with_discarded` if that is really what they want...its  dangerous to make this the default implementation as it bypasses our soft-delete.

(blocking)

Reply:
Addressed in 8911b319.